src/components/ui/Icon/README.md

```diff
@@ -0,0 +1,188 @@
+<Icon name="anchor" title="Boat anchor" />
+```
+
+## Usage
+
+### Use an `Icon` for...
+
+Use an `Icon` when you want to emphasise or clarify an important piece of text. Generally an `Icon` is used with a `Button` or `Tab` to communicate an action or navigation link, but can be used as a standalone element as well. When used as standalone elements, they are often seen in page headers, section titles, in notifications, and inline with text to add clarity.
+
+### Don't use an `Icon` for...
+
+- Displaying a complex image or photo. Use an `Image` instead.
+- Communicating a link, button, or action. Use a `Button` or `Tab` (combined with an `Icon`) instead.
+
+## Appearance
+
+### Colour
+
+An icon can be passed either a stroke or a fill colour.
+
+```jsx
+<Icon name="pen-tool" strokeColor="#053e6c" large/>
+<Icon name="target" strokeColor="#8a0000" large/>
+<Icon name="star" strokeColor="#eeb600" large/>
+<br />
+<Icon name="pen-tool" fillColor="#053e6c" large/>
+<Icon name="target" fillColor="#8a0000" large/>
+<Icon name="star" fillColor="#eeb600" large/>
+<br />
+<Icon name="pen-tool" fillColor="#5788b1" strokeColor="#043258" large/>
+<Icon name="target" fillColor="#c26767" strokeColor="#780000" large/>
+<Icon name="star" fillColor="#f7e095" strokeColor="#d7a400" large/>
+```
+
+When pairing an icon with a label, using a slightly lighter variant of the text colour helps make the two elements feel connected, whilst ensuring the icon doesn't have more visual weight than the text. Nautilus does this by default because we always want you to look good. 🙌
+
+```jsx
+import { Paragraph } from '@octopusthink/nautilus';
+
+<React.Fragment>
+  <Paragraph style={{ color: '#b32973' }}>
+    <Icon name="heart"/> I love Lucy
+  </Paragraph>
+  <Paragraph>
+    <Icon name="coffee" />
+    Lucy loves coffee
+  </Paragraph>
+</React.Fragment>
+```
+
+### Size
+
+Icons are available in five different sizes: `extraLarge` (`64rem`), `large` (`48rem`), `medium` (`24px`), `small` (`16px`), and `extraSmall` (`8px`). If you don't pass a size value, your icons will use the medium size.
+
+The default icon sizes can be tweaked in your theme. Sizes should be defined in `rem` to ensure the icons adjust to the user's font size. (The above sizes are the pixel equivalents of the `rem` sizes provided for reference.)
+
+```jsx
+<Icon name="heart" extraSmall />
+<Icon name="heart" small />
+<Icon name="heart" medium />
+<Icon name="heart" large />
+<Icon name="heart" extraLarge />
+
+<Icon name="star" extraSmall />
+<Icon name="star" small />
+<Icon name="star" medium />
+<Icon name="star" large />
+<Icon name="star" extraLarge />
+
+<Icon name="pen-tool" extraSmall />
+<Icon name="pen-tool" small />
+<Icon name="pen-tool" medium />
+<Icon name="pen-tool" large />
+<Icon name="pen-tool" extraLarge />
+```
+
+Stroke widths scale with the icon size for better legibility. Stroke widths are also customisable via your theme.
+
+[example]
```

Comment:
There's no example here. Did you have one in mind or is this a TODO?

Reply:
Fixed!